website/app_test.js

```diff
@@ -84,16 +76,6 @@ test(function createBinarySizeColumnsIrregularData() {
   assertEqual(columns, [["binary_size", 0, 0]]);
 });
 
```

Comment:
Please don't delete tests from master

Reply:
I've moved to tests in another directory. i've not deleted them, i guess?

Reply:
I mean some tests currently in masters `website/app_test.js`, I don't think they are moved to somewhere else. Looks like you rebased on master but probably unexpectedly dropped these test cases during the process

Reply:
when I had to resolve the conflict in website/app_test.js,I did not resolve it properly and it caused dropping these test cases during the process. My bad.would you please help me in resolving the conflicts?